### ISSUE TITLE
Accessibility - Metro, State, and County More Detail Buttons Not Opening Next Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Local-Labor-Market-Data-Android#22]
For some reason now when I'm on the more details button and double tap as instructed to see more details, it doesn't go to another screen. But if I move TalkBack to any other items on the same row and double tap it goes to the more details screen. The issue applies to all three detail pages (metro, state, and county).

**Comment by jfc3-dol** [comment]
I can tap on the more details button from one of the detail screens but without TalkBack on but it only works with tapping on rest of row and not the button when TalkBack is activated. Maybe the reason I can do it with TalkBack off is because of fat fingers and not tapping the button.

See screenshot with more detail button.

![bls local data - county screen -  more detail button not able to tap with talkback running](https://user-images.githubusercontent.com/10423487/58031779-1f280680-7aef-11e9-9834-96b1cfd2ee4d.jpg)

**Comment by greg-gruse** [comment]
With talkback OFF the entire row should be clicked on to go down another level.

With talkback ON you can't get to the entire row because we made all of those labels in the row clickable for individual talk back.

So with talkback ON you tap the indicator, with talk back OFF you should be able to tap anywhere in the row.

I've tested this on more recent versions of Android on my device and it works as described above.  In order to test on older versions of Android I have to use the simulator,  in this case it works as well for me but I am using a mouse with a pointer which is more exact.

Can you describe again the issue you are seeing?

**Comment by jfc3-dol** [comment]
TalkBack reads the last item at the end of the row as a button that can be tapped while the rest of the line is not. When I double tap on the button with TalkBack on it does not work for me on our Samsung Galaxy 7S running Android 7.0. The device is a year and a half maybe two years old. 

Not sure why the more details button works from the headings of the main metro, state, or county screens but not once you get to the more detailed screens. Plus, it used to work for those buttons until the previous update to this one when I created the issue.

**Comment by greg-gruse** [comment]
I'm at a loss for this issue at this point,  I've configured a simulator for a Galaxy Nexus(this is the only Galaxy simulator google provides) running Android 7.0.   The more details button works consistently in that simulator (see attached screen shot for what I'm clicking on - the green box)   could we set up a time for me to come over to your office and look at your device?

**Comment by greg-gruse** [comment]
![Screenshot_1558381152](https://user-images.githubusercontent.com/47372742/58047323-f6ffce00-7b15-11e9-9606-913f50163c8f.png)

**Comment by greg-gruse** [comment]
Also can you verify that you have the latest build that came out on Saturday?   If you go to the page in the google beta play store it should say "Last updated May 18, 2018" and it should say "OPEN" not "INSTALL".

**Comment by jfc3-dol** [comment]
I had Sarath copy the application over this morning for me as he always does. I'm assuming it's the latest version, but we can check in the morning when he's back and then I'm in too.

Typically, I'm available most weekdays from 9:30 AM - 12:30 PM and then from 1:30 PM - 5:30 PM when I go home.

I have a team meeting Tuesday mornings from 9:30 AM - 10:00 AM and then another one Friday afternoon which I'm assuming the government will possibly send people home early for the long weekend.

**Comment by jfc3-dol** [comment]
I couldn't tell looking at the application on our devices the build date.

I did notice on the "About" screen the release is 2.0, and the date listed is November 5, 2018. I'm assuming the Release date will get updated.

I am going to create an issue to make sure that gets done.

**Comment by jfc3-dol** [comment]
I am now able to use the more details button with TalkBack for the metro, state, and county detail screens.

So I'm going to close this issue.
